### DISCUSSION TITLE
Add the ability to add previously created groups to individual documents

### DISCUSSION THREAD

**Discussion by AlexShku** [outline/outline#6555] [Ideas]
Hello! 
Thank you for the wonderful 0.75.0 update. 
It would be great if it was possible to add predefined groups and not just users individually to Documents.

**Comment by AlexShku** [comment]
up

**Reply by tommoor** [reply]
https://github.com/outline/outline/issues/6639

**Comment by AlexShku** [comment]
up

**Comment by santiagobiali** [comment]
up
